### ISSUE TITLE
False Positive on library jhipster-framework-3.9.1.jar

### ISSUE THREAD

**Issue by vidhya03** [dependency-check/DependencyCheck#2999]
False positive on library jhipster-framework-3.9.1.jar -  reported as 
`cpe:2.3:a:jhipster:jhipster:3.9.1:*:*:*:*:*:*:*`

```xml
<!-- https://mvnrepository.com/artifact/io.github.jhipster/jhipster-framework -->
<dependency>
    <groupId>io.github.jhipster</groupId>
    <artifactId>jhipster-framework</artifactId>
    <version>3.9.1</version>
</dependency>
```

**Comment by jeremylong** [comment]
Due to how the NVD uses CPEs - i do not believe the CPE is wrong. However, we added a suppression rule for the CVE.

**Comment by vidhya03** [comment]
Hope this helps for other user https://github.com/jhipster/generator-jhipster/issues/11886
